_includes/release-notes/51new-features.md

```diff
@@ -195,6 +195,42 @@ Fixes for inline dialogs and menus have been included to:
 
 ## Premium Plugin changes
 
+### Advcode
```

Comment:
This should probably be `Advanced Code` or `Advanced Code Editor` I guess?

Reply:
fixed